include/language.php

```diff
@@ -702,8 +740,11 @@ public function get_search_url() {
 		if ( ( defined( 'PLL_CACHE_LANGUAGES' ) && ! PLL_CACHE_LANGUAGES ) || ( defined( 'PLL_CACHE_HOME_URL' ) && ! PLL_CACHE_HOME_URL ) ) {
 			/**
 			 * Let's use `site_url()` so the returned URL will be filtered properly according to the current domain.
+			 *
+			 * @phpstan-var non-empty-string $search_url
 			*/
-			return site_url( set_url_scheme( $this->search_url, 'relative' ) );
+			$search_url = site_url( set_url_scheme( $this->search_url, 'relative' ) );
+			return $search_url;
```

Comment:
Same as above.

Reply:
Done in [c104c0d](https://github.com/polylang/polylang/pull/1207/commits/c104c0d85a346f61dbd3e971bf85b29b5c312c0f).